### ISSUE TITLE
Program crashes a _LOT_ on OS X 10.7.4

### ISSUE THREAD

**Issue by spyder-issues-migrator** [spyder-ide/spyder#1042]
_From [lancebo...@qwest.net](https://code.google.com/u/115604211761405613769/) on 2012-05-23T06:08:39Z_

Spyder Version:  2.1.9
Python Version:  2.7.3
Qt Version:      4.7.4, PyQt4 (API v2) 4.9.1 on Darwin 

What steps will reproduce the problem?

 1.Use the program 2. 3. What is the expected output? What do you see instead? Expected: Useful work accomplished.
instead: Crash. 

Please provide any additional information below

. More to follow.

_Original issue: http://code.google.com/p/spyderlib/issues/detail?id=1042_

**Comment by spyder-issues-migrator** [comment]
_From [pierre.raybaut](https://code.google.com/u/pierre.raybaut/) on 2012-05-23T04:44:29Z_

I'm afraid we won't be able to do much without more details on these crashes you are referring to.

**Comment by spyder-issues-migrator** [comment]
_From [petrus.h...@gmail.com](https://code.google.com/u/113904071220560210639/) on 2012-05-24T05:41:39Z_

Maybe try to upgrade qt, I had some performance issues when using 4.7.

I'm using:
Python 2.7.3, Qt 4.8.2, PyQt4 (API v2) 4.9.1 on Darwin

qt, python and pyqt, Installed using the homebrew installer. Daily usage, no crashes lately.

**Comment by spyder-issues-migrator** [comment]
_From [ccordoba12](https://code.google.com/u/ccordoba12/) on 2012-07-07T10:08:59Z_

Without further info we can't do anything else. Please report it again if you can give us an idea of what's happening.

**Status:** Invalid

**Comment by spyder-issues-migrator** [comment]
_From [lancebo...@qwest.net](https://code.google.com/u/115604211761405613769/) on 2012-07-11T14:48:21Z_

Sorry for the crappy bug report.

I am mostly evaluating the program and using it for light work. The first few times I used it, I got many crashes, and mostly in the first seconds after launching and beginning work. I thought it might be Qt-related--I feel that the everything-in-one-window GUI design is an inefficient use of screen space and i work on a laptop. Thus, I normally try to undock the pseudo-windows (I don't know what they are called) into actual windows that can overlap. My hunch was that it was related to undocking these windows. However, more recent sessions with the program have resulted in no crashes. I have no idea what the reason could be for the change in stability. I built from MacPorts not long before I filed the bug report and have not rebuilt since.

So, once again, sorry for the drive-by bug report. OK to close it.

Jerry
